src/lib/config.js

```diff
@@ -206,9 +206,11 @@ function getLocalConfig () {
   expiry.maxHoldTime = +Config.getEnv(envPrefix, 'MAX_HOLD_TIME') || DEFAULT_MAX_HOLD_TIME
 
   // The spread is added to every quoted rate
-  const fxSpread = Number(Config.getEnv(envPrefix, 'FX_SPREAD')) || DEFAULT_FX_SPREAD
+  const fxSpreadString = Config.getEnv(envPrefix, 'FX_SPREAD')
+  const fxSpread = fxSpreadString ? +fxSpreadString : DEFAULT_FX_SPREAD
```

Comment:
I took this from the master branch: https://github.com/interledger/five-bells-connector/blob/master/src/lib/config.js#L214-L218. Basically manually cherrypicking. I am happy to change this to `Number()` but it would cause a conflict if we merge this to `master`.